pkg/cmd/server/api/v1/conversions.go

```diff
@@ -19,6 +19,9 @@ func init() {
 			if obj.ServingInfo.RequestTimeoutSeconds == 0 {
 				obj.ServingInfo.RequestTimeoutSeconds = 60 * 60
 			}
+			if obj.ServingInfo.MaxRequestsInFlight == 0 {
+				obj.ServingInfo.MaxRequestsInFlight = 500
```

Comment:
We make more calls in more loops - 100 is the constant factor for that extra work.

> On Jun 15, 2015, at 10:51 PM, Jordan Liggitt <notifications@github.com> wrote:
> 
> In pkg/cmd/server/api/v1/conversions.go:
> 
> > @@ -19,6 +19,9 @@ func init() {
> >  			if obj.ServingInfo.RequestTimeoutSeconds == 0 {
> >  				obj.ServingInfo.RequestTimeoutSeconds = 60 * 60
> >  			}
> > +			if obj.ServingInfo.MaxRequestsInFlight == 0 {
> > +				obj.ServingInfo.MaxRequestsInFlight = 500
> I question the legitimacy of that tuning, given that nothing was being limited by this. Also, watches aren't supposed to fall under this
> 
> —
> Reply to this email directly or view it on GitHub.
>